src/sonic-host-services/scripts/caclmgrd

```diff
@@ -273,6 +274,42 @@ class ControlPlaneAclManager(daemon_base.DaemonBase):
 
         return allow_internal_docker_ip_cmds
 
+    def get_loopback_ip_from_loopback_table(self, loopback_table, loopback_name):
```

Comment:
i think the name is not aligned. I suggest define this as 'get_ip_from_loopback_table' and just pass loopback_name, if function is for loopback table. If not, rename to get_ip_from_interface_table' and pass table and name. Suggest to make it generic.

Reply:
fixed